server/security.js

```diff
@@ -14,7 +14,8 @@ let defaultCSPDirectives = {
   ],
   frameSrc: [
     "'self'",
-    "https://docs.google.com"
+    "https://docs.google.com",
+    "blob:"
```

Comment:
http://stackoverflow.com/questions/30023608/how-to-use-frame-src-and-child-src-in-firefox-and-other-browsers seems to indicate that `frameSrc` is right (again)?  I could do both.